plugins/module_utils/dnac.py

```diff
@@ -2160,13 +2160,26 @@ def compare_list(list1, list2):
 def fn_comp_key(k, dict1, dict2):
     return dnac_compare_equality(dict1.get(k), dict2.get(k))
 
+def normalize_ipv6_address(ipv6):
+    """
+    Normalize an IPv6 address for consistent comparison.
+    """
+    try:
+        return str(ipaddress.IPv6Address(ipv6))
+    except ValueError:
```

Comment:
Can we print a log message?

        self.log("Invalid IPv6 address: {}".format(ipv6))

Reply:
Done